sources/platform/integrations/keboola.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Keboola
+sidebar_lbel: Keboola
+description: Learn how to integrate your Apify datasets with Airbyte.
+sidebar_position: 11.16
+slug: /integrations/keboola
+---
+
+**Integrate your Apify Actors with Keboola.**
```

Comment:
I would describe Keboola itself, how it work and for what you can use it.
You can reuse some text from https://help.apify.com/en/articles/2003234-keboola-integration

Reply:
expanded the intro hook slightly